File: Test1_solution/solution_8.py

def consecutive(arr,n):
    current_count=0
    max_count=0
    
    for i in range(0,n):
        if(arr[i]==1):
            current_count+=1
        else:
            current_count=0
    max_count=max(current_count,max_count)
    return max_count

arr=[1,1,0,1,1,1]
print("max count of consecutive one=",consecutive(arr,len(arr)))

File: Test1_solution/solution_8.java
public class solution_8
{
    public static void main(String args[])
    {
        int arr[]={1,1,0,0,1,1,1};
        System.out.println("consecutive ones="+consecutive_one(arr,arr.length));
    }
    public static int consecutive_one(int arr[],int n)
    {
        int count=0;
        int max_count=0;
        for(int i=0;i<n;i++)
        {
            if(arr[i]==1)
                count++;
            else
                count=0;
            
            if(max_count<count)
            {
                max_count=count;
            }
        }
        return max_count;
    }
}